.github/BOTMETA.yml

```diff
@@ -591,6 +592,9 @@ files:
     labels:
       - cloud
     support: community
+  $module_utils/hpe3par.py: &hpe3par
```

Comment:
It should be `$module_utils/storage/hpe3par.py: &hpe3par`

Reply:
Whoops, I made a mistake a `hpe3par` directory is missing, it should be: `$module_utils/storage/hpe3par/hpe3par.py`.

I created a PR which document `module_utils` tree recommendation: #45561.